### DISCUSSION TITLE
Cool site to get shadcn-ui themes from

### DISCUSSION THREAD

**Discussion by corysimmons** [shadcn-ui/ui#2735] [Show and tell]
https://ui.jln.dev

The dev just added 1000 popular VSCode themes to it as well (which actually look really good in shadcn-ui)! 💪

![image](https://github.com/shadcn-ui/ui/assets/224636/c219fc79-c496-4e6e-a2b5-1cd59f4517bb)

![image](https://github.com/shadcn-ui/ui/assets/224636/4e4afcaf-36e4-4fec-a487-335bee1af74f)

Just find a theme you like, then copy/paste the css vars! Now your site looks different from everyone else.

![image](https://github.com/shadcn-ui/ui/assets/224636/5f15cfd3-0db5-43f7-acf0-f1a26b5c1298)

**Comment by jln13x** [comment]
thanks for sharing @corysimmons 🙌

**Comment by gutorskiNovobit** [comment]
Bit Defender blocks the link with a phishing warning

**Reply by jln13x** [reply]
any idea whats causing this?

**Reply by mxfng** [reply]
Can't open the link either

**Reply by jln13x** [reply]
> Can't open the link either

Mind sharing a screenshot?

**Reply by mxfng** [reply]
Sure.

I got this issue:
```
An error occurred: | SSL_ERROR_RX_RECORD_TOO_LONG
```

![Screenshot 2025-06-02 at 7 28 25 PM](https://github.com/user-attachments/assets/a2fbb7d4-778b-41fc-9118-48deb762ac59)
